server/views/components/actions-card/actionsCard.test.ts

```diff
@@ -0,0 +1,36 @@
+import nunjucks from 'nunjucks'
+import * as cheerio from 'cheerio'
+import type { Action } from 'viewModels'
+
+nunjucks.configure([
+  'node_modules/govuk-frontend/dist/',
+  'node_modules/@ministryofjustice/frontend/',
+  'server/views/',
+  __dirname,
```

Comment:
Includes set up so that the import in the test should have the same path as using the real component. Hopefully makes it easy to crib off when you want to use the component.